### PR TITLE
Add linting to travis

### PR DESCRIPTION
**Pull request by wtrocki**
## Description
Adding ts-lint to be run as travis build. 

## Progress
- [x] Added lint to travis

### CODE REVIEW

**Comment by paolobueno** [comment]
so from failed build, we need to add `tslint` and maybe `typescript` to devDependencies

**Comment by paolobueno** [comment]
The npm lint command could also be changed to `tslint '*/*/src/**/*.ts' '*/*/test/**/*.ts'` (not sure if it supports two globs actually.)

Also it might be better to remove the `--fix` flag, because we want the job to error so the developer run --fix locally and then commit.

**Comment by wtrocki** [comment]
@paolobueno 👍  Applied your comments and fixed current lint issues :D

File: demo/server/src/index.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ function onError(error: any) {
  * Event listener for HTTP server "listening" event.
  */
 function onListening() {
-  var addr = server.address();
+  const addr = server.address();
+  // tslint:disable-next-line:no-console
```
Comment (JameelB) [inline demo/server/src/index.ts]: remove comment

**Pull request merged**